response_operations_ui/controllers/message_controllers.py

```diff
@@ -53,19 +69,37 @@ def get_conversation_count(survey_id, conversation_tab, business_id):
         raise ApiError(response)
 
     logger.info("Count successful")
+
     try:
-        return response.json()['total']
+        totals = response.json()['totals']
+
+        # Secure Message uses different identifiers to the tab names used in the ui, this translates the names
+        totals['initial'] = totals.pop('new_respondent_conversations')
+        totals['my messages'] = totals.pop('my_conversations')
+
+        totals['current'] = totals[conversation_tab]
+
+        return totals
     except KeyError:
-        logger.exception("Response was successful but didn't contain a 'total' key")
+        logger.exception("Response was successful but didn't contain a 'totals' key")
```

Comment:
A keyError isn't just a missing totals key.  It could also be a missing key in the totals dictionary.  This needs to be made more clear

Reply:
After the change for the previous comment it can only be a missing totals key.